.editorconfig

```
@@ -1,22 +1,19 @@
 root = true
 
 [*]
-charset=utf-8
+charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+end_of_line = lf
 indent_style = space
 indent_size = 2
 
-[*.{py,sh,js,ps1}]
+[*.{py,sh,ps1,js,json,css}]
```

```suggestion
[*.{py,sh,ps1,js,json}]
```

There aren't any `.css` files checked into source control here.

```shell
git ls-files | grep css
```

But there can sometimes be some laying around in your local development environment after rendering the sphinx docs or pkgdown docs.

Given that they're auto-generated... I don't think we want to generate any linter warnings for such files.

---

I added `css` for possible **future** css files in our repo. But I didn't think about the case you described. I think it makes more sense than mine.